openmc/mgxs/mgxs.py

```diff
@@ -214,6 +231,28 @@ def __deepcopy__(self, memo):
         else:
             return existing
 
+    def _add_angle_filters(self, filters):
+        # Add the azimuthal and polar bins if needed.
+        if self.num_polar or self.num_azimuthal:
+            # Then the user has requested angular data, so create the bins
+            if self.num_polar:
+                pol_bins = np.linspace(0., np.pi, num=self.num_polar + 1,
```

Comment:
Even without the unity part, I'd still have to have the `pol_bins` and `azi_bins` `linspace` commands as well as the filter inserts.  Instead of doing that for all the filters in the method I figured I'd let a private method do it for me.  (Again this relates to I want to avoid passing these filters to OpenMC unless the actual filters are required)